libs/SalesforceAnalytics/build.gradle

```diff
@@ -46,7 +48,7 @@ android {
   }
   defaultConfig {
     testApplicationId "com.salesforce.androidsdk.analytics.tests"
-    testInstrumentationRunner "com.salesforce.androidsdk.analytics.util.test.JUnitReportTestRunner"
+    testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
```

Comment:
We don't need our custom test runner anymore - it has been around for a while for legacy reasons. In the new framework you can specify annotations that define the size of the test and the framework will batch them accordingly.